README.md

```diff
@@ -86,7 +86,7 @@ The Azure MCP Server provides tools for interacting with the following Azure ser
 - List Log Analytics workspaces
 - Query logs using KQL
 - List available tables
-- Configure monitoring options
+- Configure monitoring optionsqw
```

Comment:
```suggestion
- Configure monitoring options
```

Reply:
@hallipr can you add cspell to this repo like the azure-sdk repos have enabled?